chains/v20/chains.json

```diff
@@ -107,6 +107,12 @@
                     "type": "subquery",
                     "url": "https://subquery-governance-polkadot-prod.novasama-tech.org"
                 }
+            ],
+            "referendum-summary": [
+                {
+                    "type": "novasama",
+                    "url": "https://opengov-backend-stg.novasama-tech.org/not-secure/api/v1/referendum-summaries/list"
```

Comment:
need to remove `not-secure/`

Reply:
I haven't changed it on stg yet, we can keep it as it is and move to prod version, when it will be deployed